.github/workflows/ci-image-dev.yml

```diff
@@ -17,18 +17,31 @@ on:
 
 jobs:
   push-to-ecr:
-    uses: Zilliqa/gh-actions-workflows/.github/workflows/ci-dockerized-app-build-push-ecr.yml@main
-    with:
-      commitOrTag: ${{ inputs.commitOrTag }}
-      file: docker/Dockerfile
-      context: ''
-      image-name: zilliqa
-      cache-key: zilliqa
-      runner: "ubuntu-latest-4-cores"
-    secrets:
-      registry: ${{ secrets.AWS_ACCOUNT_ID_ZILLIQA }}.dkr.ecr.${{ secrets.AWS_REGION_ZILLIQA }}.amazonaws.com
-      registry-username: ${{ secrets.ECR_USERNAME }}
-      registry-password: ${{ secrets.ECR_PASSWORD }}
-      aws-region: ${{ secrets.AWS_REGION_ZILLIQA }}
-      cache-username: ${{ secrets.CACHE_USERNAME }}
-      cache-password: ${{ secrets.CACHE_PASSWORD }}
+    name: build
+    runs-on: docker
```

Comment:
Is this a self-hosted runner?

Reply:
yes, I avoided to use the label self-hosted because it is default for every self hosted runner